plugins/org.eclipse.xtext.xtext.ui/src/org/eclipse/xtext/xtext/ui/wizard/project/XtextProjectCreator.java

```diff
@@ -234,25 +238,57 @@ public GradleContributor(ProjectDescriptor descriptor) {
 		@Override
 		public void contributeFiles(IProject project, IFileCreator fileWriter) {
 			fileWriter.writeToFile(
-				"{\n" +
-				"	\"1.0\": {\n" +
-				"		\"project_path\": \""+ getLogicalPath() + "\",\n" +
-				"		\"project_dir\": \""+ descriptor.getLocation() + "\",\n" +
-				"		\"connection_project_dir\": \""+ descriptor.getConfig().getParentProject().getLocation() + "\",\n" +
-				"		\"connection_gradle_distribution\": \"GRADLE_DISTRIBUTION(WRAPPER)\"\n" +
-				"	}\n" +
-				"}\n",
-			".settings/gradle.prefs");
+				"connection.arguments=\n" +
+				"connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)\n" +
+				"connection.gradle.user.home=null\n" +
+				"connection.java.home=null\n" +
+				"connection.jvm.arguments=\n" +
+				"connection.project.dir=" + getConnectionLogicalPath() +"\n" +
+				"derived.resources=" + getDerivedResources() + "\n" +
```

Comment:
@oehme do you mean the locations? So I can simply not put any location here? Or even completely remove the properties connection.project.dir and derived.resources?